plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -353,8 +353,8 @@ def main():
 
         user_group_id = group.id
     else:
-        user = gitlab_instance.users.list(username=gitlab_instance.user.username)[0]
-        user_group_id = user.id
+        namespace = gitlab_instance.namespaces.list(search=gitlab_instance.user.username)[0]
+        user_group_id = namespace.id
 
     if not user_group_id:
         module.fail_json(msg="Failed to find the user/group id which required to find namespace")
```

Comment:
```suggestion
        module.fail_json(msg="Failed to find the namespace or group ID which is required to look up the namespace")
```

Reply:
I changed all of them to namespace_id, because that name makes sense for both personal projects as well as group projects